app/views/kits/_form.html.erb

```diff
@@ -0,0 +1,48 @@
+<%= simple_form_for @kit, remote: request.xhr?, html: { class: 'form-horizontal' } do |f| %>
+  <section class="content">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-12">
+          <div class="card card-primary">
+            <div class="card-body">
+              <%= f.input :name, label: "Name", wrapper: :input_group do %>
+                <span class="input-group-text"><i class="fa fa-tag"></i></span>
+                <%= f.input_field :name, class: "form-control" %>
+              <% end %>
+
+              <% if current_user.organization_admin? %>
+                <%= f.input :name, label: "On hand minimum quantity", wrapper: :input_group do %>
+                  <%= f.input_field :on_hand_minimum_quantity, input_html: {value: 0}, class: "form-control" %>
+                <% end %>
+                <%= f.input :name, label: "On hand recommended quantity", wrapper: :input_group do %>
+                  <%= f.input_field :on_hand_recommended_quantity, class: "form-control" %>
+                <% end %>
```

Comment:
These could probably be struck -- it doesn't look like those fields exist for kits

Reply:
@armahillo Ah, thanks for the catch! I'll fix it. 👀